client/packages/common/src/ui/layout/tables/utils/ColumnDefinitionSetBuilder.ts

```diff
@@ -84,12 +83,7 @@ const getColumnLookup = <T extends RecordWithId>(): Record<
     key: 'expiryDate',
     label: 'label.expiry',
     width: 110,
-    formatter: dateString => {
-      if (dateString === '[multiple]') return '[multiple]';
-      return dateString
-        ? Formatter.expiryDate(new Date(dateString as string)) || ''
-        : '';
-    },
+    format: ColumnFormat.Date,
```

Comment:
Displaying in full for now until confirmation by @mark-prins

Reply:
yes, in full thanks 🙏